rest/utilities_testing.go

```diff
@@ -1386,6 +1386,31 @@ func (bt *BlipTester) SendRev(docId, docRev string, body []byte, properties blip
 
 }
 
+// Expects adminChannels of the form `admin_channels":["alpha"]`
+// If scope and collection are non-zero, builds collection access string for the collection
+func AdminChannelGrant(collection *db.DatabaseCollection, adminChannels string) (collectionAdminChannels string) {
+	collectionName := collection.Name()
+	scopeName := collection.ScopeName()
+	if base.IsDefaultCollection(scopeName, collectionName) {
+		return adminChannels
+	}
+
+	return fmt.Sprintf(`"collection_access":{%q: {%q: {%s}}}`, scopeName, collectionName, adminChannels)
+}
+
+func GetNamedDatastore(dataStores []sgbucket.DataStoreName, collection *db.DatabaseCollection) sgbucket.DataStoreName {
```

Comment:
If this is needed, I feel like collection should just implement the sgbucket.DataStoreName interface.